src/SDKs/DevTestLabs/Management.DevTestLabs/Properties/AssemblyInfo.cs

```diff
@@ -7,7 +7,7 @@
 [assembly: AssemblyTitle("Microsoft Azure DevTest Labs Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure DevTest Labs Resources.")]
 [assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.2.0.0")]
+[assembly: AssemblyFileVersion("2.0.0.0")]
```

Comment:
@ryanvog any reason you want AssemblyFileVersion major version to be higher than Assembly version?
This looks like a breaking change, please make AssemblyVersion to go to 2.0 also